src/easylink/implementation_metadata.yaml

```diff
@@ -39,6 +39,15 @@ step_4a_python_pandas:
   outputs:
     step_4a_main_output: result.parquet
 step_4b_python_pandas:
+  steps: 
+    - step_4b
+  image_path: /mnt/team/simulation_science/priv/engineering/er_ecosystem/images/r-image.sif
+  script_cmd: Rscript /dummy_step.R
+  env:
+    INPUT_ENV_VARS: "DUMMY_CONTAINER_MAIN_INPUT_FILE_PATHS,DUMMY_CONTAINER_SECONDARY_INPUT_FILE_PATHS"
+  outputs:
+    step_4b_main_output: result.parquet
+step_4b_python_r:
```

Comment:
This isn't strictly necessary; I just wanted to remind myself how the imlementation_metadata spec worked